### PR TITLE
Ensure hive resources when installing via hive

### PR DESCRIPTION
**Pull request by s-amann**
installation is finished to give time for the API server certificate to be updated.

### Which issue this PR addresses:

Fixes https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/16592928

### What this PR does / why we need it:

Adds a step before install is complete for when we install via HIVE the kubeconfig in HIVE for the cluster is updated.

### Test plan for issue:

manual

### Is there any documentation that needs to be updated for this PR?

no, we need to create some detailed HIVE on ARO documentation.

### CODE REVIEW

File: pkg/cluster/install.go [hunk 1]
```diff
@@ -301,20 +301,25 @@ func (m *manager) Install(ctx context.Context) error {
 			steps.Action(m.configureIngressCertificate),
 			steps.Condition(m.ingressControllerReady, 30*time.Minute, true),
 			steps.Action(m.configureDefaultStorageClass),
-			steps.Action(m.finishInstallation),
```
Comment (jhoreman) [inline pkg/cluster/install.go:304]: Hey Spencer, thanks for this. Can you help me understand why we're removing the finishInstallation call here when the function still exists on line 366

Comment (jhoreman) [reply pkg/cluster/install.go:304]: After further review, I see it on line 311 now as a subsequent step. Thanks!

**Review by jhoreman — APPROVED** [review]

**Review by SudoBrendan — APPROVED** [review]
This LGTM, the only thing I'd ask is what manual steps were run to test? Adding that as breadcrumbs for later will be helpful. Thanks, and great work!

**Review by cblecker — APPROVED** [review]

**Comment by s-amann** [comment]
closing in favor of #2602

**Pull request closed without merging**